src/agent/command_handler/include/command_handler.hpp

```diff
@@ -91,6 +91,15 @@ namespace command_handler
 
                 if (cmd.value().ExecutionMode == module_command::CommandExecutionMode::SYNC)
                 {
+                    if (cmd.value().Module == "restart")
```

Comment:
Is this block if necessary? This is already done as part of the dispatch command set in https://github.com/wazuh/wazuh-agent/pull/386/files#diff-da3d5e8dd4752020c509b5b337eb7254314aad465439e743138acdb52912a34cR171.